includes/class-main.php

```
@@ -18,988 +18,993 @@
+				if ( ! in_array( $handle, $this->exclude_delay_js, true ) ) {
+					$tag = str_replace( ' src', ' wppo-src', $tag );
+					$tag = preg_replace(
+						'/type=("|\')text\/javascript("|\')/',
+						'type="wppo/javascript" wppo-type="text/javascript"',
+						$tag
+					);
+				}
+			}
+
+			return $tag;
 		}
```

PHPMD (2.15.0)

[error] The method add_defer_attribute is not named in camelCase. (undefined)

(CamelCaseMethodName)